### DISCUSSION TITLE
Script

### DISCUSSION THREAD

**Discussion by sarjugarg** [jhuckaby/Cronicle#594] [Q&A]
Hi,

i want to use this tool for other purpose.

To remotely start/stop/check Linux process and schedule them thru GUI. I do not need to login on remote servers to do these operation

This is a fantastic tool for replacing cron entries but we cannot manage process remotely. Check part can be done by other monitoring tools like Zabbix

is there any plugin etc for this

Regards
Sarju

**Comment by jhuckaby** [comment]
Try using the Shell Plugin to SSH to your remote servers and run the commands you want there.

**Reply by chandrabhan81** [reply]
is it possible to execute without worker server. and is there any option to stop completed job.

**Comment by sarjugarg** [comment]
Thanks.

but how do I see option on GUI to start/stop the process/job...

**Comment by jhuckaby** [comment]
While a job is running, click the "Abort Job" button in the top-right corner, or use the [abort_job](https://github.com/jhuckaby/Cronicle/blob/master/docs/APIReference.md#abort_job) API call.

This will send a `SIGTERM` signal to the shell (assuming you're using the Shell Plugin), which will propagate it to the SSH command, and that *should* also terminate the remote script (I think).
